src/core.test.ts

```diff
@@ -359,4 +359,34 @@ describe('Core', () => {
     await core.disablePerLine(results, ['test/ban-nullish-coalescing-operator']);
     expect(await readFile(iff.paths['src/ban-nullish-coalescing-operator.js'], 'utf-8')).toMatchSnapshot();
   });
+  test.runIf(ESLint.version.startsWith('9.'))('supports unstable_config_lookup_from_file flag', async () => {
```

Comment:
If `unstable_config_lookup_from_file` is stabilized in a new version of v9, this test will break. I think it would be best to fix it when it breaks. There is no need to do anything at this time.